### DISCUSSION TITLE
Coalition submarine ambush gives money

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#13989] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The coalition low reputation submarine ambush `coalitionattack1` gives money while separatist doesnt

![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/b548141d-e38e-45c4-87cd-5d3745d479a0)

### Reproduction steps

1. setreputation coalition -100
2. undock and redock until coalition sub spawns
3. kill enemy sub
4. dock
5. get money

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/63f3ff0662cea2a1463d30911b617a3e3b9ef61c, fix coming up in the next unstable build!
